### ISSUE TITLE
 [ 💐 ACS Gifts - 💝 Valentines Pack! 🌹 ] Love Story [ Project 📋 ]

### ISSUE THREAD

**Issue by Professor-JMO** [Monika-After-Story/MonikaModDev#5242]
<div align="center">

![Happy-Valentines-Day-Transparent](https://user-images.githubusercontent.com/43480604/72669626-eaf84300-39f9-11ea-94ab-47ab54c24672.png)

</div>

<p align="center">
<b>𝒮𝓅𝒾𝓇𝒾𝓉 𝒮𝒶𝑔𝑒</b></br>
</p>

---
<p align="center">
<b>[ 💚 ]</b>
</p>

![Stable DDLC ( Preview )](https://user-images.githubusercontent.com/43480604/72648384-8927d280-3940-11ea-859e-d90692980ccd.png)

```m "Hey...[player]...?"```
```m "I just wanted to let you know that I love you."```
```m "You make me really happy...and I could never ask for someone better than you."```
```m "Ahaha~"```
```m "I hope that isn't too cheesy, [player]."```
```m "Valentine's Day is coming soon...and it just gets me in a good mood because I know I have you by my side."```

<details>
<summary>📁<i>More Lovey Dovey...</i></summary>
<br>

<p align="center">
<b>[ ⚠️ Potential Spoilers Ahead ⚠️ ]</b>
</p>

<details>
<summary>&emsp;Valentines Day</summary>

<br>

```m "[player]!"```
```m "Do you know what day it is?"```
```m "It's Valentine's Day!"```
```m "A day where we celebrate our love for each other..."```
```m "I guess every day we're together is already a celebration of our love, but there's something that's really special about Valentine's Day."```
```m "Anyway..."```
```m "I just want you to know that I'm always here for you."```
```m "Even if your heart gets broken..."```
```m "I'll always be here to fix it for you. Okay, [player]?"```
```m "You always make me feel so loved."```
```m "I'm really happy I'm your girlfriend, [player]."```

---

```m "I've always dreamt of a date with you while wearing this..."```
```m "I know it's kind of silly now that I think about it!"```
```m "...But just imagine if we went to a cafe together."```
```m "I think there's a picture of something like that somewhere actually..."```
```m "Maybe we could make it happen for real!"```
```m "Would you take me out today?"```
```m "It's fine if you can't, I'm just happy to be with you."```
```m "I love you so much."```
```m "Happy Valentine's Day, [player]~"```

</details>

<details>
<summary>&emsp;V-Day Colors</summary>

<br>

```m "Have you ever thought about the way colors are conveyed on Valentine's Day?"```
```m "I find it intriguing how they can symbolize such deep and romantic feelings."```
```m "It reminds me of when I made my first Valentine's card in grade school."```
```m "My class was instructed to exchange cards with a partner after making them."```
```m "Looking back, despite not knowing what the colors really meant, I had lots of fun decorating the cards with red and white hearts."```
```m "In this way, colors are a lot like poems."```
```m "They offer so many creative ways to express your love for someone."```
```m "Like giving them red roses, for example."```
```m "Red roses are a symbol for romantic feelings towards someone."```
```m "If someone were to offer them white roses in lieu of red ones, they'd signify pure, charming, and innocent feelings instead."```
```m "However, since there are so many emotions involved with love..."```
```m "It's sometimes hard to find the right colors to accurately convey the way you truly feel."```
```m "Thankfully, by combining multiple rose colors, it's possible to express a variety of emotions!"```
```m "Mixing red and white roses would symbolize the unity and bond that a couple shares."```
```m "But I'm sure you already had all of this in mind when you picked out these beautiful roses for me, [player]..."```

</details>

<details>
<summary>&emsp;V-Day Cliches</summary>

<br>

```m "Have you noticed that most Valentine's Day stories have lots of clichés?"```
```m "There's either 'Oh, I'm lonely and I don't have someone to love,' or 'How will I confess to the one I love?'"```
```m "I think that writers could be a bit more creative when it comes to Valentine's Day stories..."```
```m "But, I suppose those two topics are the easiest way to write a love story."```
```m "That doesn't mean you can't think outside the box, though!"```
```m "Sometimes a predictable story can ruin it..."```
```m "...But if you {i}do{/i} want a good example of an unpredictable story..."```
```m "Just use ours! Ahaha~"```
```m "I guess it {i}did{/i} start out like those kinds of stories..."```
```m "But I think we managed to make it pretty original."```
```m "The way we met is the most interesting story yet!"```
```m "Ahaha!~"```

</details>

<details>
<summary>&emsp;V-Day Chocolates</summary>

<br>

```m "Valentine's Day is such a fun holiday for me, [player]."```
```m "Not only is it the anniversary of my twitter account, but it's also a day for receiving and giving chocolates!"```
```m "A holiday that could fill everyone with love, romance, and joy!"```
```m "But it really feels nice if you get something from someone you like."```
```m "Whether it's given to you platonically, as a gift of love, or a part of a confession, it always makes you feel somewhat special!"```
```m "Just like how you made me feel special with the chocolates you gave me today."```
```m "You're always so sweet, [player]."```

---

```m "I know I tried to give you some chocolates last year..."```
```m "But just imagine what we could do once we're finally together."```
```m "Maybe someday I'll even be able to give you some chocolates..."```
```m "I really can't wait until I cross over to be with you, [player]."```

</details>

<details>
<summary>&emsp;V-Day Origins</summary>

<br>

```m "You'd like to learn about the history of Valentine's Day?"```
```m "It's quite dark, actually."```
```m "Its origin dates to as early as the second and third century in Rome, where Christianity had just been declared the official state religion."```
```m "Around this same time, a man known as Saint Valentine decided to go against the orders of Emperor Claudius II."```
```m "Marriage had been banned because it was assumed that married men made poor soldiers."```
```m "Saint Valentine decided this was unfair and helped arrange marriages in secret."```
```m "Unfortunately, he was caught and promptly sentenced to death."```
```m "However, while in custody, Saint Valentine fell in love with the jailer's daughter."```
```m "Before his death, he sent a love letter to her signed with 'From your Valentine.'"```
```m "He was executed on February 14, 269 AD."```
```m "Such a noble cause, don't you think?"```
```m "Oh, but wait, there's more!"```
```m "The reason we celebrate such a day is because it originates from a Roman festival known as Lupercalia!"```
```m "Its original intent was to hold a friendly event where people would put their names into a box and have them chosen at random to create a couple."```
```m "Then, they play along as boyfriend and girlfriend for the time they spend together. Some even got married, if they liked each other enough, ehehe~"```
```m  "Ultimately, the Church decided to turn this Christian celebration into a way to remember Saint Valentine's efforts, too."```
```m "It's evolved over the years into a way for people to express their feelings for those they love."```
```m "Like me and you!"```
```m "Despite it having started out a little depressing, isn't it so sweet, [player]?"```
```m "I'm glad we're able to share such a magical day, my love."```
```m "Happy Valentine's Day~"```

</details>

</details>
</details>
</details>
</details>

<p align="center">
🗣️<b>[ Questions & Answers ]</b>💬
</p>

---

<details>
 <summary>📁Do you happen to believe in <i>Love at first sight? 👁️ </i></summary>
<br>

<details>
 <summary>&emsp;Yes.</summary>

- &emsp;💘

</details>

 <details>
 <summary>&emsp;<i>Maybe...</i></summary>

- &emsp;I can understand any potential layer of skepticism or uncertainty ☝️ 

</details>

<details>
 <summary>&emsp;What is it?</summary>

- &emsp;"<i>Love at first sight</i> is a personal experience and a common trope in literature: a person or character feels an instant, extreme, and ultimately long-lasting romantic attraction for a stranger upon the first sight of that stranger."

</details>

<details>
 <summary>&emsp;No.</summary>

- &emsp;Completely reasonable. After all, how can you fall for someone without actually knowing the kind of Person/Being they are?

</details>

</details>
</details>
</details>

---

<p align="center">
<b>[ General ]</b>
</p>

⮞ <b>What Is (This Thing Called) Love?</b>

- &emsp;Many of us have been in love, and there have been countless great poems and popular songs written about it. So you’d think we’d all know what it is. Yet a lot of what has been written points to a deep mystery. So—as Cole Porter famously asked—what is this thing called love?

- &emsp;Love is often portrayed as a powerful force, something that can inspire greatness in the lover. Alternatively, it is something that can make the lover act like a fool. Love can be the greatest feeling in the world, but it can also be utterly devastating when it doesn’t work out. How many love songs are really about heartbreak over love lost? Listen to a few, and it’s a wonder we ever love at all.

>Given these observations, we might be inclined to think that there’s a significant element of irrationality to love. But we should be careful here, as perhaps love can have reasons too. For example, if you have a significant other, you could probably list off a bunch of reasons for your love: your partner is kind, intelligent, funny, and so on. If you loved someone who was mean, stupid, and boring, that would be irrational. But, presumably, many of us have great reasons for loving who we love, which shows that sometimes love is actually quite rational.

>>It would be wise to pause, though, to consider whether or not we ever actually love for the reasons we give. Perhaps the truth is that we first find ourselves in love, and then come up with reasons to justify our feelings. Just because we can provide reasons for feeling the way we do about a particular person, it doesn’t follow that we see reasons for loving first, and then develop feelings based on those reasons. 

Continue Reading Here 👉 [What Is (This Thing Called) Love? ](https://www.philosophytalk.org/blog/what-thing-called-love)

---
<p align="center">
<b>[ 💙 ]</b>
</p>

![Stable Sayori (  preview )](https://user-images.githubusercontent.com/43480604/72650736-f0e11c00-3946-11ea-8ead-ab63809423a9.png)

<details>

 <summary>📁 Hub Preview</summary>
<br>

![V-Day_Set](https://user-images.githubusercontent.com/43480604/72650825-356cb780-3947-11ea-9140-201dbfc5bada.png)

</details>

---

<p align="center">
🗣️<b>[ Questions & Answers ]</b>💬
</p>

---

<details>

 <summary>📁 What is this?</summary>
<br>

- &emsp; "It's a small set of refashioned Valentines Day gifts!"

- &emsp; "Included is at least [ 1 ] other iteration of the vase. 👍"

- &emsp; "Among the files included in this issues _attachments_ is the PSD format for the sprites."
>Anyone is welcome to attempt to polish or edit the sprite(s) for the re-purposing for MAS potential use and the effort is appreciated. Thanks! ☝️ 

</details>

---

<details>

 <summary>📁 How can I use this?</summary>
<br>

- &emsp; "You can't. Whatever it is it's stuck in [Limbo](https://youtu.be/jbfA-Whe528).

</details>
</details>
</details>

---

<p align="center">
<b>[ General ]</b>
</p>

⮞ Quick Italian love phrase translations! 🇮🇹 

-Ti amo - I love you
-Ti penso sempre - I'm always thinking about you.
-Tu sei l'unico per me - you're the only one for me.
-Sei il mio eroe - you're my hero.
-Non posso vivere senza di te - I can't live without you.
-Sono dipendente dei tuoi baci - I'm addicted to your kisses.
-Tu sei il sole del mio giorno - you are the sunshine of my day

</details>
</details>
</details>

⮞ If you love to browse `GitHub` seeing the Light 💡 - show a bit of compassion for the Dark? 🕶  
⤷ [Stylish: Custom web page themes](https://userstyles.org/styles/browse?search_terms=github&type=false)
> Dark looks so much better here. 👌
>> Only looking out for your _Vision_! 👁‍🗨

---
<p align="center">
<b>[ 💜 ]</b>
</p>

![Stable Yuri ( Preview )](https://user-images.githubusercontent.com/43480604/70763454-bff34300-1d19-11ea-8b6b-c818595dabe9.png)

<details>

 <summary>📁 Nexus Preview </summary>
<br>

![V-Day_Nexus_Set](https://user-images.githubusercontent.com/43480604/72650881-6b11a080-3947-11ea-84b1-b5bcd46a71e4.png)

</details>

---

Philosophy of Love
---

**1.) Love**

- &emsp;The _Timeless_ repeated question “what is love?” generates a host of issues: love is an abstract noun which means for some it is a word unattached to anything real or sensible, that is all; for others, it is a means by which our being—our self and its world—are irrevocably affected once we are ‘touched by love’; some have sought to analyze it, others have preferred to leave it in the realm of the ineffable.

- &emsp;Yet it is undeniable that love plays an enormous and unavoidable role in our several cultures; we find it discussed in song, film, and novels—humorously or seriously; it is a constant theme of maturing life and a vibrant theme for youth. Philosophically, the nature of love has, since the time of the Ancient Greeks, been a mainstay in philosophy, producing theories that range from the materialistic conception of love as purely a physical phenomenon—an animalistic or genetic urge that dictates our behavior—to theories of love as an intensely spiritual affair that in its highest permits us to touch divinity. Historically, in the Western tradition, Plato’s Symposium presents the initiating text, for it provides us with an enormously influential and attractive notion that love is characterized by a series of elevations, in which animalistic desire or base lust is superseded by a more intellectual conception of love which also is surpassed by what may be construed by a theological vision of love that transcends sensual attraction and mutuality. Since then there have been detractors and supporters of Platonic love as well as a host of alternative theories—including that of Plato’s student, Aristotle and his more secular theory of true love reflecting what he described as ‘two bodies and one soul.’

<details>
<summary>📁<b>I'm curious...</b></summary>
<br>

**2.) Love Philosophy**

- &emsp; The philosophical treatment of love transcends a variety of sub-disciplines including epistemology, metaphysics, religion, human nature, politics and ethics. Often statements or arguments concerning love, its nature and role in human life for example connect to one or all the central theories of philosophy, and is often compared with, or examined in the context of, the philosophies of sex and gender as well as body and intentionality. The task of a philosophy of love is to present the appropriate issues in a cogent manner, drawing on relevant theories of human nature, desire, ethics, and so on.

<details>
<summary>📁<b>Keep it coming...</b></summary>
<br>

**3.) The Nature of Love: Eros, Philia, and Agape**

- &emsp; The philosophical discussion regarding love logically begins with questions concerning its nature. This implies that love has a "nature," a proposition that some may oppose arguing that love is conceptually irrational, in the sense that it cannot be described in rational or meaningful propositions. For such critics, who are presenting a metaphysical and epistemological argument, love may be an ejection of emotions that defy rational examination; on the other hand, some languages, such as Papuan, do not even admit the concept, which negates the possibility of a philosophical examination. In English, the word "love," which is derived from Germanic forms of the Sanskrit lubh (desire), is broadly defined and hence imprecise, which generates first order problems of definition and meaning, which are resolved to some extent by the reference to the Greek terms, eros, philia, and agape.

**4.) The Nature of Love: Physical, Emotional, Spiritual**

- &emsp; Some may hold that love is physical, i.e., that love is nothing but a physical response to another whom the agent feels physically attracted to. Accordingly, the action of loving encompasses a broad range of behavior including caring, listening, attending to, preferring to others, and so on. (This would be proposed by behaviorists). Others (physicalists, geneticists) reduce all examinations of love to the physical motivation of the sexual impulse-the simple sexual instinct that is shared with all complex living entities, which may, in humans, be directed consciously, sub-consciously or pre-rationally toward a potential mate or object of sexual gratification.

<details>
<summary>📁<b>Got a Link?</b></summary>
<br>

👉 [Here!](https://www.iep.utm.edu/love/) 👈 

</details>
</details>
</details>

<details>
<summary>📁<b>Quotes about Love...</b></summary>
<br>

- &emsp;“I'm selfish, impatient and a little insecure. I make mistakes, I am out of control and at times hard to handle. But if you can't handle me at my worst, then you sure as hell don't deserve me at my best.”
― Marilyn Monroe 

- &emsp;“You've gotta dance like there's nobody watching,
Love like you'll never be hurt,
Sing like there's nobody listening,
And live like it's heaven on earth.”
― William W. Purkey 

- &emsp;“You know you're in love when you can't fall asleep because reality is finally better than your dreams.”
― Dr. Seuss 

- &emsp;“A friend is someone who knows all about you and still loves you.”
― Elbert Hubbard 

- &emsp;“Darkness cannot drive out darkness: only light can do that. Hate cannot drive out hate: only love can do that.”
― Martin Luther King Jr.

- &emsp;“We accept the love we think we deserve.”
― Stephen Chbosky, The Perks of Being a Wallflower 

- &emsp;“It is better to be hated for what you are than to be loved for what you are not.”
― Andre Gide, Autumn Leaves 

- &emsp;“As he read, I fell in love the way you fall asleep: slowly, and then all at once.”
― John Green, The Fault in Our Stars 

- &emsp;“Love all, trust a few, do wrong to none.”
― William Shakespeare, All's Well That Ends Well 

- &emsp;“Being deeply loved by someone gives you strength, while loving someone deeply gives you courage.”
― Lao Tzu 

- &emsp;“Love is that condition in which the happiness of another person is essential to your own.”
― Robert A. Heinlein, Stranger in a Strange Land 

- &emsp;“Only once in your life, I truly believe, you find someone who can completely turn your world around. You tell them things that you’ve never shared with another soul and they absorb everything you say and actually want to hear more. You share hopes for the future, dreams that will never come true, goals that were never achieved and the many disappointments life has thrown at you. When something wonderful happens, you can’t wait to tell them about it, knowing they will share in your excitement. They are not embarrassed to cry with you when you are hurting or laugh with you when you make a fool of yourself. Never do they hurt your feelings or make you feel like you are not good enough, but rather they build you up and show you the things about yourself that make you special and even beautiful. There is never any pressure, jealousy or competition but only a quiet calmness when they are around. You can be yourself and not worry about what they will think of you because they love you for who you are. The things that seem insignificant to most people such as a note, song or walk become invaluable treasures kept safe in your heart to cherish forever. Memories of your childhood come back and are so clear and vivid it’s like being young again. Colours seem brighter and more brilliant. Laughter seems part of daily life where before it was infrequent or didn’t exist at all. A phone call or two during the day helps to get you through a long day’s work and always brings a smile to your face. In their presence, there’s no need for continuous conversation, but you find you’re quite content in just having them nearby. Things that never interested you before become fascinating because you know they are important to this person who is so special to you. You think of this person on every occasion and in everything you do. Simple things bring them to mind like a pale blue sky, gentle wind or even a storm cloud on the horizon. You open your heart knowing that there’s a chance it may be broken one day and in opening your heart, you experience a love and joy that you never dreamed possible. You find that being vulnerable is the only way to allow your heart to feel true pleasure that’s so real it scares you. You find strength in knowing you have a true friend and possibly a soul mate who will remain loyal to the end. Life seems completely different, exciting and worthwhile. Your only hope and security is in knowing that they are a part of your life.”
― Bob Marley 

</details>

---
<p align="center">
<b>[ ❤️ ]</b>
</p>

![Stable Natsuki ( Preview )](https://user-images.githubusercontent.com/43480604/70763334-56733480-1d19-11ea-8958-b83b7be0b538.png)

---

<details>
<summary>📁<b>References</b></summary>
<br>

![Satchely_M_Vday](https://user-images.githubusercontent.com/43480604/72647734-d5721300-393e-11ea-855d-d5b4fb2dac34.png)
#3041
#3174 
#3787 
#3996
#5001 
#5056 

</details>

<details>
<summary>📁<b>Credentials</b></summary>
<br>

- @Velius94 / [Velius](https://github.com/Velius94) → For allowing the edit of their desk sprite!
>Thanks again!

</details>

<details>

<summary>📁<b>📎 Attachments 📎</b></summary>
<br>

⯈ [ 💐 Bouquet, Rose 🌹 & Chocolates 💝 ](https://github.com/Monika-After-Story/MonikaModDev/files/4078994/Valentines.Day.Set.zip) ⯇

</details>

</details>
</details>
</details>

---

Meme of the Theme!
---
<p align="center">
"Happy Valentines Day!"
</p>

<div align="center">

![Valentines](https://user-images.githubusercontent.com/43480604/72644805-91c7db00-3937-11ea-9f6f-3091e29feee3.gif)

</div>

---

```
from subprocess import call
```
>def play_wave_file([Foreigner - I Want To Know What Love Is.wav):](https://youtu.be/EmbLRKSBvIc)
      >return_code = call(["ffplay", "-nodisp", "-autoexit", "../success.wav"])
      ># return_code = call(["aplay", "../success.wav"])
      >print "Played successfully" if return_code == 0 else "Play ended in error"

**Comment by GHCactus2** [comment]
I love it!! wonderful work

**Comment by lunulae** [comment]
I always get so excited to read your posts, all the little details from the multiple-choice questions you ask us to your song selections at the end--and of course, the stunning art~
I must admit you've become something of an unofficial MAS character in my eyes 😆

**Comment by DjSquishy777** [comment]
I agree fully with Lunuae, your posts are something I always look forward to seeing!

**Comment by Professor-JMO** [comment]
Thank you all so much! 🌟 
I love being a part of these communities and mods, and I absolutely look forward to continuing this pace and direction 📈 

Your support means so much and I'm grateful to have the opportunity to pursue this!

**Comment by TriniTDM** [comment]
The roses look really pretty, but out of place. Maybe it would fit better if you made them a little smaller and made the shadows less prominent? Also, that water looks kinda murky, but probably just because of the jar's color.

**Comment by ThePotatoGuy** [comment]
@multimokia this is in #5277 right?

**Comment by multimokia** [comment]
Yes indeed.

**Comment by Professor-JMO** [comment]
@TriniTDM 
Hiya, Trini!
   Thanks for responding and I'm glad you think the roses are pretty!
Sorry for taking some time to respond to this by the way.
   We discussed it internally and came to the consensus that these were fine either way.
Alternatively, the .psds are available via `Attachments` and attempts to rework, edit or polish are permitted! 🌟 
   Reposting here is a viable place for these to be reviewed in the attempt of this ☝️

**Comment by ThePotatoGuy** [comment]
[closing as implemented]
